keycloak/keycloakify/LICENSE

```diff
@@ -1,7 +1,5 @@
 MIT License
 
-This license applies specifically to this directory

```

Comment:
(good to revert this :) )

Reply:
oh yes :D 

811b519e21a9dbf37648bbd0b0a51b065587e31d